Prac_05/wimbledon.py

```diff
@@ -0,0 +1,51 @@
+FILENAME = "wimbledon.csv"
+
+
+def main():
+    """Wimbledon.py main function."""
+    records = get_records(FILENAME)
+    countries, champion_to_times_won = process_data(records)
+    print_results(countries, champion_to_times_won)
+
+
+def get_records(filename):
+    """Gets record from file."""
+    records = []
+    with open(filename, "r", encoding="utf-8-sig") as in_file:
+        in_file.readline()  # Remove header from columns
+        for line in in_file:
+            parts = line.strip().split(",")
+            records.append(parts)
+    return records
+
+
+def process_data(records):
+    """Processes data into dictionary and set."""
+    countries = set()
+    champion_to_times_won = {}
+    count = 0
+    for record in records:
+        countries.add(record[1])
+        champion = record[2]
```

Comment:
These are looking like "magic numbers", maybe you could make them constants to improve readability?